docs/current_docs/manuals/developer/functions.mdx

```diff
@@ -49,27 +128,30 @@ The result will look something like this:
 }
 ```
 
-## Class methods
-
-Dagger Functions are implemented as [@dagger.function][@function] decorated
-methods, of a [@dagger.object_type][@object_type] decorated class.
-
-It's possible for a module to implement multiple classes (*object types*), but
-**the first one needs to have a name that matches the module's name**, in
-*PascalCase*. This object is sometimes referred to as the *main object*.
+## Arguments 
```

Comment:
I made this h4 to exclude from toc since its specific to python - but open to other ideas